pkg/framework/aries/default.go

```diff
@@ -122,7 +147,64 @@ func setAdditionalDefaultOpts(frameworkOpts *Aries) error {
 		frameworkOpts.msgSvcProvider = &noOpMessageServiceProvider{}
 	}
 
-	// TODO add SecretLock creation here.. #1150
+	return nil
+}
+
+func createDefSecretLock(opts *Aries) error {
+	keySize := sha256.Size
+
+	file, err := os.OpenFile(defMKPath, os.O_WRONLY|os.O_CREATE, 0600)
```

Comment:
we are hardcoding a file lookup as the default secret lock?
this will break WASM.

Reply:
Also, in combination with the other default - local DB storage, this default local file secret lock is arguably not adding much.

Reply:
another option would be to set an `env` variable, but it will be erased when the Go program shuts down. 

This means a new master key is generated on each aries.New() call without KMS/SecretLock options which makes keys stored in a local DB storage unlockable.

Reply:
I will move this function to default_srv.go and use an the env reader master key in default_js_wasm.go

Reply:
@Baha-sk There are many options as SecretLock is an injectable interface. The problem is really that there might not be any acceptable default at the framework level (other than no-op).

There may be acceptable defaults for a particular target like cmd/agent-rest & the JS/WASM worker in cmd.

Reply:
sure, I'll add noop secretlock and make it as a framework default.

other specific targets may have their own secret lock instances (part of other PRs)..

I'll push another change in a bit...

Reply:
updated with noop SecretLock